src.ts/tests/ffi.spec.ts

```diff
@@ -229,3 +323,11 @@ describe('bindings', () => {
     }).toThrow('Expected 32 bytes, got 1')
   })
 })
+
+function createScalarFromIndex(index: number): Uint8Array {
+  const BYTES_PER_SCALAR = 32
+
+  const scalar = new Uint8Array(BYTES_PER_SCALAR)
+  scalar[0] = index // Set first byte to index (little-endian)
+  return scalar
+}
```

Comment:
We could use 31 bytes and have random elements, since 31 bytes will always fit within a scalar (shouldn't matter for the tests just making a note)